es6-shim.js

```diff
@@ -1096,7 +1098,9 @@
 
   var ArrayPrototypeShims = {
     copyWithin: function copyWithin(target, start) {
-      var end = arguments[2]; // copyWithin.length must be 2
+      // copyWithin.length must be 2, so we can't add `end` to the arguments
```

Comment:
Well, I was just repeating and expanding the comment which was there originally, which was trying to explain why this couldn't be simply declared as `copyWithin(target, start, end)`.  But in theory our test cases should actually be verifying the length property of the function, so the comment should be unnecessary...